### ISSUE TITLE
Welcome back! apt-cyg!!!

### ISSUE THREAD

**Issue by kou1okada** [transcode-open/apt-cyg#73]
I congratulate return of the apt-cyg. :-)

**Comment by transcode-open** [comment]
Thank you!

**Comment by l0ki000** [comment]
welcome back!

**Comment by mosbth** [comment]
Nice to see you again!

**Comment by idarlund** [comment]
Welcome back :)

**Comment by brianmorton** [comment]
Glad you're back!

**Comment by boothj5** [comment]
Just another quick thanks for all your work up till now.

**Comment by JoshuaKolden** [comment]
👍

**Comment by Mparaiso** [comment]
Glad to see the project back !

**Comment by jaimerod** [comment]
+1

**Comment by alphapapa** [comment]
I hesitate to mention this, but I think the truth is important, so...

The facts as I understand them are:

1.  Steven Penny forked the project to svnpenn/sage.
2.  Steven Penny issued an illegal DMCA takedown notice to GitHub ("under penalty of perjury"), claiming ownership of the original copyright, causing this repo to be taken down.  He commented on his own repo, "The apt-cyg repository has been closed by me under a DMCA takedown, likely permanently. I am seeking a court order today to assist in this."
3.  Steven Penny changed the license on his fork from Stephen Jungels's MIT license to GPL.
4.  Steven Penny removed [the required copyright notice](https://github.com/transcode-open/apt-cyg/blob/master/apt-cyg#L15) from his fork, violating Stephen Jungels's license.
5.  Stephen Jungels filed a DMCA counter-claim with GitHub, causing this repo to be restored.

Since then, Steven Penny has banned from his repo anyone who created an issue merely politely asking about the license status and the relation of the fork to the original repo.  Regarding the license, he stated on his repo, "I dont really care about this."

A good summary of these events is found at http://archive.is/k7sLb

Given these events, it is clear which fork is hostile and which developer is dishonest.  And therefore, it is clear which repo should be avoided by users who desire to run software from trustworthy, honest developers.

As much as I'm sure Mr. Jungels wants to move on--and as much as I respect him for doing so, despite having every right to pursue the matter further as the aggrieved party--at the same time I think it's important, for users' sake, that these events be documented.  The sage repo and project should be avoided at all costs.  And if this is not documented prominently, unsuspecting users may discover it and begin using it, putting themselves at risk.

**Comment by boothj5** [comment]
I'm glad you mentioned this @alphapapa I had no idea, and it's pretty shocking behaviour.

**Comment by samalloing** [comment]
Can you remove/change the bountysource link in the readme, because now I donated to the wrong person :-(

**Comment by svnpenn** [comment]
https://github.com/transcode-open/apt-cyg/issues/74

**Comment by sn-donbenjamin** [comment]
Wow guys this is tough to follow.
I don't think I can add anything useful except this.
Who's ever code I forked its very useful. I thank them for sharing it.
Open source is just that we have to be respectful of the person who designed it
anyone not doing that should be banned from here. Using some DCMA BS to hack someone
else's work is really a pity since we are all here to benefit from others sharing, knowledge and experiences.

**Comment by RyuDragonRyder** [comment]
Okay, just to make sure. I am on the right fork for apt-cyg, right?

**Comment by ghost** [comment]
@RyuDragonRyder this project is inactive - i have a similar project here

https://github.com/cup/pear

**Comment by mosbth** [comment]
@RyuDragonRyder the project seems inactive, but its the original repo for apt-cyg and the script is still working as one could expect.
Works for me.

**Comment by mrjoshuak** [comment]
@RyuDragonRyder yes transcode-open is the originator of the project.

However, Steven Penny did most of the work on apt-cyg.  Which is probably why he felt he had the right to claim licensing authority over it. It was a bad move based on a misunderstanding of how open source licensing works, but that doesn’t change the fact that Steven Penny did most of the work.  Now that he’s gone not much is happening here. 

That being said, it’s a pretty simple script. It shouldn’t be too hard for people to pick up the ball and submit pull requests.

**Comment by cup** [comment]
@mrjoshuak yes that is an insightful comment. I think you are the first person in 3 years who realizes the truth of the situation - rather than jumping to conclusions.

I would only disagree on the point of pull request - as the owner of this repo has made it pretty clear that he doesnt wish to maintain any more - and he doesnt wish to transfer the repo either. trust me, i tried. at any rate, i can entertain issues or pull request at my repo - if anyone can get past the bad history

**Comment by gavenkoa** [comment]
For wandering people: [list of  forks](https://github.com/transcode-open/apt-cyg/network/members) suggests that following repo is heir of this project:

https://github.com/kou1okada/apt-cyg

It has 317 commit, when this repo has only 155.
